### ISSUE TITLE
Why features loss is using detach

### ISSUE THREAD

**Issue by jeff39389327** [somuchtome/SimAC#3]
loss = loss + target_loss.detach().item()
I cant understand  why using the detach  not<  loss + target_loss  >

**Comment by somuchtome** [comment]
> loss = loss + target_loss.detach().item() I cant understand why using the detach not< loss + target_loss >


Sorry for the misunderstanding. Thank you for your feedback!

**Comment by jeff39389327** [comment]
Based on the GitHub code execution results using the detached version, the output results are close to the paper's data. I want to ask in CVPR 2024 SimAC, whether the actual implementation uses detach or not?

**Comment by SignedQiu** [comment]
The same issue.
